### PR TITLE
Fix misc send issues

### LINKED ISSUE #915: Token balance validation off by one

### PR DESCRIPTION
**Pull request by HenryNguyen5**
Fixes 
#915 
#914 
#745

### CODE REVIEW

File: webpack_config/makeConfig.js [hunk 1]
```diff
@@ -295,11 +296,11 @@ module.exports = function(opts = {}) {
     filename: options.isProduction ? '[name].[chunkhash:8].js' : '[name].js',
     publicPath: isDownloadable && options.isProduction ? './' : '/',
     crossOriginLoading: 'anonymous'
-  }
-
+  };
 
   // The final bundle
   return {
+    devtool,
```
Comment (wbobeirne) [inline webpack_config/makeConfig.js]: For anyone else reviewing: This is the only non-formatting change.

Can't believe I missed this D:

Comment (HenryNguyen5) [reply webpack_config/makeConfig.js]: ah yeah, looks like we dont have prettier setup for things outside of `common`

File: common/components/AmountFieldFactory/AmountInputFactory.tsx [hunk 2]
```diff
@@ -11,23 +11,23 @@ interface OwnProps {
...
+    const { currentValue, onChange, withProps, isNonStandard } = this.props;
 
     return (
       <Query
         params={['readOnly']}
         withQuery={({ readOnly }) =>
           withProps({
             currentValue,
-            isValid: validAmount,
+            isValid: !!currentValue.value || isNonStandard,
```
Comment (wbobeirne) [inline common/components/AmountFieldFactory/AmountInputFactory.tsx]: It took me a while to figure out that `currentValue.value` is null if it exceeds the balance. This is a little unintuitive IMO. What do we do to get the value field's value in the non-standard transactions?

Comment (HenryNguyen5) [reply common/components/AmountFieldFactory/AmountInputFactory.tsx]: https://github.com/MyEtherWallet/MyEtherWallet/wiki/Transaction-Validity-Spec
^ For checking for non-standard transactions

Comment (HenryNguyen5) [reply common/components/AmountFieldFactory/AmountInputFactory.tsx]: And yeah, i hear ya on the null value, https://github.com/MyEtherWallet/MyEtherWallet/issues/901
The initial thought process behind it was to make it impossible to send a transaction if a field was invalid ( as the transaction would have null fields and then be rejected by a node). But this lead to the inability to set custom error messages.

**Pull request merged**